lang/en.json

```
@@ -0,0 +1,12 @@
+{
+  "Slack": {
+    "ChannelID": "Slack Channel ID",
```

Nitpick: I think it's more consistent to use `Id` instead of `ID` in names.
```suggestion
    "ChannelId": "Slack Channel ID",
```